mempool/src/shared_mempool/types.rs

```diff
@@ -40,7 +40,7 @@ where
 {
     pub mempool: Arc<Mutex<CoreMempool>>,
     pub config: MempoolConfig,
-    pub network_senders: HashMap<NodeNetworkId, MempoolNetworkSender>,
+    pub network_senders: HashMap<NetworkId, MempoolNetworkSender>,
```

Comment:
similarly, we should create an aggregate `MempoolNetworkSender` that takes `PeerNetworkId` and routes it to the corresponding network sender

Reply:
or are you planning to do that in the `NetworkInterface`?

Reply:
Planning on doing something with it and the `NetworkInterface` to consolidate these.